port.h

```diff
@@ -105,4 +105,13 @@ SOFTWARE.
 
 #endif
 
+// Check for 'thread_local' specifier support
```

Comment:
Nit: Period at the end. As per Google C++ Style Guide, comments must be complete sentences. Thanks!